cluster/examples/kubernetes/ceph/upgrade-from-v1.0-apply.yaml

```diff
@@ -135,3 +135,51 @@ rules:
       - create
       - update
       - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: rook-ceph-system
+  labels:
+    operator: rook
+    storage-backend: ceph
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - pods
+      - configmaps
+      - services
+    verbs:
+      - get
+      - list
+      - watch
+      - patch
+      - create
+      - update
+      - delete
+  - apiGroups:
+      - apps
+    resources:
+      - daemonsets
+      - statefulsets
+      - deployments
+    verbs:
+      - get
+      - list
+      - watch
+      - create
+      - update
+      - delete
+  - apiGroups:
+      - policy
+    resources:
+      - poddisruptionbudgets
```

Comment:
`poddisruptionbudgets` is requested as part of the `rook-ceph-global-rules` ClusterRole above in this file. Why do we need to request it again here? I'm wondering about the rest of this update as well. Maybe it's related to the aggregate rules that were added in 1.0 which may be causing an issue for you

Reply:
I can only say, that adding `poddisruptionbudgets` helped me to make operator working again.

Reply:
@DeamonMV do you have the error message that showed up when it wasn't there?